WordPress/src/main/java/org/wordpress/android/ui/accounts/LoginEpilogueActivity.java

```diff
@@ -0,0 +1,39 @@
+package org.wordpress.android.ui.accounts;
+
+import org.wordpress.android.R;
+import org.wordpress.android.util.ToastUtils;
+
+import android.os.Bundle;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v7.app.AppCompatActivity;
+
+public class LoginEpilogueActivity extends AppCompatActivity implements LoginEpilogueFragment.LoginEpilogueListener {
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+
+        setContentView(R.layout.login_epilogue_activity);
+
+        if (savedInstanceState == null) {
```

Comment:
Just out of curiosity, why do we need a fragment here? It seems like the activity could do all the work.